include/outcome/experimental/status_result.hpp

```diff
@@ -28,7 +28,11 @@ Distributed under the Boost Software License, Version 1.0.
 #include "../basic_result.hpp"
 #include "../policy/fail_to_compile_observers.hpp"
 
+#if !OUTCOME_USE_SYSTEM_STATUS_CODE && __has_include("status-code/include/system_error2.hpp")
```

Comment:
I wonder, should we not remove the macro and instead probe for has include of `"./status-code/include/system_error2.hpp` preferentially, and only if not found, try including `<status-code/system_error2.hpp?`? In other words, pin the preference to any locally installed copy?

Reply:
I have no strong feelings one way or the other, _but_ I'd like to point out that 
> pin the preference to any locally installed copy

is not fully guaranteed by branching on `__has_include("status-code/...")` alone. `status-code` uses "simple" include guards which implies that the selection depends on the include order within the TU.

Reply:
That's a reasonable argument. Ok, accepted.